files/zh-cn/learn/forms/styling_web_forms/index.md

```diff
@@ -159,7 +159,7 @@ textarea {
 
 ![This is what we want to achieve with HTML and CSS](/files/4149/screenshot.png)
```

Comment:
Noticed that there are a lot of `/files/` images causing 404, we may open another PR to deal with these instances.